webpack.config.js

```diff
@@ -19,4 +19,7 @@ module.exports = {
       },
     ],
   },
+  resolve: {
+    extensions: ['*', '.js', '.jsx'],
```

Comment:
Oh, seems like you did it! Means you should be able to remove all the .jsx from your imports too.